Azure/sonic-buildimage
# Fixing some python errors in the common PDDF platform classes

#### Why I did it
- Some imports were missing in the common class which could cause errors when some of the functions of the class are invoked.
- Py2->Py3 migration related change was missing in one of the class.

#### How I did it
- Adding proper imports
- Correcting the subprocess call as per Py3

#### How to verify it
- Load the image on a platform using PDDF. Check the logs if there is any python backtrace pointing to '/usr/local/lib/python3.9/dist-packages/sonic_platform_pddf_base/pddfapi.py'. There should not be any.
- Run the class methods invoking functions from un-imported modules.
```
root@sonic:/home/admin# python3
Python 3.9.2 (default, Feb 28 2021, 17:03:44)
[GCC 10.2.1 20210110] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from sonic_platform import platform
>>> ch = platform.Platform().get_chassis()
>>> ch.get_thermal(0).set_high_threshold(80)
True
>>>
```
#### Which release branch to backport (provide reason below if selected)



These fixes are required to go in 202111 branch.

#### Description for the changelog


#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
